examples/README.md

```diff
@@ -0,0 +1,36 @@
+This folder contains an example configuration file, as well as necessary files to run a demo of Transition. This demo corresponds to the territory of the Réseau de transport de Longueuil area.
+
+# Running the demo
+
+1. Make sure all the pre-requisites are installed. For complete installation instructions of all dependencies, see the [instructions for Ubuntu](../../docs/transition/installationUbuntu20.04.md).
+
+2. Update the .env file and set the `PROJECT_CONFIG` to point to the config file in this directory
```

Comment:
Update or copy the .env.example file??

Reply:
Maybe say "Or set the environment variable..." ????
or is this too much details/options?

Reply:
I assume they followed the previous instructions in their current state, so they have a .env at this point. I haven't tested with just setting environment variable, and had bad luck when trying with `yarn start:json2capnp`, so I wouldn't say it here. I know that's what you do in terraform. But if you're reading this, you want it as simple as possible.